.github/actions/env/action.yml

```diff
@@ -24,22 +24,23 @@ runs:
         auto-update-conda: true
         python-version: ${{ inputs.python-version }}
 
-    - if: ${{ contains(fromJson('["lts", "nightly"]'), inputs.pytorch-version ) }}
-      name: Install dependencies
+    - name: Install curl via conda
       shell: bash -l {0}
-      run: |
-        conda install curl -c conda-forge
-        conda install pytorch cpuonly -c pytorch-${{ inputs.pytorch-version }}
-        pip install .[dev,x]
+      run: conda install curl -c conda-forge
 
+    - name: Install Kornia dev
+      shell: bash -l {0}
+      run: pip install -e .[x,dev]
+
+    - if: ${{ contains(fromJson('["lts", "nightly"]'), inputs.pytorch-version ) }}
+      name: Install pytorch
+      shell: bash -l {0}
+      run: conda install pytorch cpuonly -c pytorch-${{ inputs.pytorch-version }}
```

Comment:
now that i see this, @lferraz was complaining the other day that we install many time torch ? via conda and pip ?
https://github.com/kornia/kornia/blob/master/setup_dev_env.sh#L67

also in there idk why we have torchvision, i don't think is needed